src/components/dashboard/index.js

```diff
@@ -256,6 +257,7 @@ const Dashboard = (props) => {
     return (
         <div ref={dashboardEl} id={baseId} className={classes.gridRoot}>
             {!userProfile?.id && <Banner />}
+            {<UserSurvey />}
```

Comment:
If it's much of a bother then don't, but if it's easy, it would be nice to let this card be hidden by users if they've already done the survey.

Reply:
That will be nice but this there is not an easy way to track if the user has taken the survey and I don't users to simply hide and not take it.